core-bundle/src/Command/UserListCommand.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\Model\Collection;
+use Contao\UserModel;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Lists Contao back end users.
+ *
+ * @internal
+ */
+class UserListCommand extends Command
+{
+    protected static $defaultName = 'contao:user:list';
+
+    /**
+     * @var ContaoFramework
+     */
+    private $framework;
+
+    public function __construct(ContaoFramework $framework)
+    {
+        $this->framework = $framework;
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addOption('column', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'The columns display in the table')
+            ->addOption('admins', null, InputOption::VALUE_NONE, 'Return only admins')
+            ->addOption('format', null, InputOption::VALUE_REQUIRED, 'The output format', 'text')
+            ->setDescription('Lists Contao back end users.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if ($input->getOption('admins')) {
+            $io->note('Only showing admin accounts');
+        }
+
+        $users = $this->getUsers($input->getOption('admins'));
+
+        if (null === $users) {
+            $io->error('No accounts found.');
+
+            return 1;
```

Comment:
I don't think this is an error. Also, it should display an empty array in JSON format.

Reply:
Correct. Fixed in e4ea80b10b2d7cee15f76d7a6680084afdfea3c0.